.github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml

```diff
@@ -43,6 +42,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Set up sentry cli
```

Comment:
Do we want issues sent to Sentry during integration tests?
I suspect not, but if so, I'm interested in a understanding the reason.

Reply:
Duplicate comment